go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	cosmossdk.io/math v1.3.0
 	github.com/cometbft/cometbft v0.38.9
```

Comment:
CometBFT [v.038.10](https://github.com/cometbft/cometbft/releases/tag/v0.38.10) just came out as well.

Reply:
@insumity After some testing I learned that this bump does not work with cosmos-sdk and ibc-go because none they havent upgraded to use it. Additionally, hermes encounters weird deserialization errors, even on the latest version (v1.10.0). This was discovered during e2e testing.

We cannot bump comet at this point.